src/overlays/actors/ovl_En_Hs2/z_en_hs2.c

```diff
@@ -27,25 +29,132 @@ const ActorInit En_Hs2_InitVars = {
     (ActorFunc)EnHs2_Update,
     (ActorFunc)EnHs2_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hs2/EnHs2_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Hs2/EnHs2_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x11, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 40, 40, 0, { 0, 0, 0 } },
+};
+
+static Vec3f D_80A6F4CC = { 300.0f, 1000.0f, 0.0f };
+
```

Comment:
(you dont have to do this if you dont want, but)
since this is only relevant for one function and the order works out, you can make this in-function-static if you want. if you go with it, you can keep it at the top of the list above the other stack variables